### PR TITLE
Added PHP Nette Framework

### PR DESCRIPTION
**Pull request by n0nag0n**
Added the PHP Nette Framework.

### CODE REVIEW

File: frameworks/PHP/nette/benchmark_config.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+		"fortune_url": "/database/fortune",
+        "port": 8080,
+        "approach": "Realistic",
+        "classification": "Fullstack",
+        "database": "mysql",
+        "framework": "Nette",
+        "language": "PHP",
+        "flavor": "None",
+        "orm": "Full",
+        "platform": "None",
```
Comment (joanhey) [inline frameworks/PHP/nette/benchmark_config.json:20]: `"platform": "FPM/FastCGI",`

File: frameworks/PHP/nette/benchmark_config.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        "webserver": "nginx",
+        "os": "Linux",
+        "database_os": "Linux",
+        "display_name": "Nette",
+        "notes": "",
+        "versus": "None"
```
Comment (joanhey) [inline frameworks/PHP/nette/benchmark_config.json:26]: `"versus": "php"`

To check the framework overhead

File: frameworks/PHP/nette/benchmark_config.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+		"query_url": "/database/query?queries=",
+		"update_url": "/database/update?queries=",
+		"fortune_url": "/database/fortune",
+        "port": 8080,
+        "approach": "Realistic",
+        "classification": "Fullstack",
+        "database": "mysql",
+        "framework": "Nette",
+        "language": "PHP",
+        "flavor": "None",
```
Comment (joanhey) [inline frameworks/PHP/nette/benchmark_config.json:18]: `"flavor": "PHP8",`

**Review by joanhey — CHANGES_REQUESTED** [review]

File: frameworks/PHP/nette/benchmark_config.json (hunk 2 above)
Comment (n0nag0n) [reply frameworks/PHP/nette/benchmark_config.json]: Done, thanks

File: frameworks/PHP/nette/benchmark_config.json (hunk 1 above)
Comment (n0nag0n) [reply frameworks/PHP/nette/benchmark_config.json]: Done, thanks

File: frameworks/PHP/nette/benchmark_config.json (hunk 3 above)
Comment (n0nag0n) [reply frameworks/PHP/nette/benchmark_config.json]: Done, thanks

**Review by joanhey — APPROVED** [review]

**Comment by n0nag0n** [comment]
Side note, with PHP are you guys allowing opcache.preload scripts or are you trying to compare all frameworks against not preloading scripts? [https://www.php.net/manual/en/opcache.preloading.php](https://www.php.net/manual/en/opcache.preloading.php)

**Comment by joanhey** [comment]
You can use preload script.
But with PHP8.1, the difference is minimal

Some frameworks us it, and others not.

File: frameworks/PHP/nette/composer.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
+{
+	"name": "nette/web-project",
+	"description": "Nette: Standard Web Project",
+	"keywords": ["nette"],
+	"type": "project",
+	"license": ["MIT", "BSD-3-Clause", "GPL-2.0", "GPL-3.0"],
```
Comment (joanhey) [inline frameworks/PHP/nette/composer.json:6]: Is it correct all these licenses togheter ?

**Comment by joanhey** [comment]
You can delete the `robots.txt` file. It isn't use it in the bencmark.

File: frameworks/PHP/nette/composer.json (hunk 4 above)
Comment (joanhey) [reply frameworks/PHP/nette/composer.json:6]: With the `GPL` you can't create proprietary apps !!!

Comment (n0nag0n) [reply frameworks/PHP/nette/composer.json:6]: Yes they actually let you choose your license. https://github.com/nette/application/blob/master/license.md

**Comment by n0nag0n** [comment]
Deleted the file......forgot to use [ci skip] :(

**Comment by joanhey** [comment]
Are you familiar with Laminas framework?

Because my implementation, still don't work. :(

**Comment by n0nag0n** [comment]
I'll be honest, this was a beast to get working in the vagrant box, but I finally got everything working and running. I'm familiar with Laminas but not in building with Laminas.

**Comment by joanhey** [comment]
OK
But you know Fat-Free
I need a hand with Fat-free.  Send me a mail.

**Comment by n0nag0n** [comment]
Sent ya an email

**Comment by joanhey** [comment]
#7703
:+1:

**Comment by joanhey** [comment]
Nette average very bad in this benchmark.
It's normal after read that:
https://forum.nette.org/en/21784-persistent-connection-to-mysql-database

I had a large discussion, for the same, with Doctrine.
Now they fix it.

**Comment by n0nag0n** [comment]
Yes I noticed the same. I did ask a question and then @rixafy on discord let me know if one change I can make. https://forum.nette.org/en/35661-techempower-benchmarks-needs-help I can make the one change but it may need more help

**Pull request merged**